src/kernel/capinv.c

```diff
@@ -15,6 +15,7 @@
 #include "include/tcap.h"
 #include "include/chal/defs.h"
 #include "include/hw.h"
+#include "include/chal/chal_prot.h"
```

Comment:
I remember we talked about renaming it to `chal_proto.h`.

Reply:
renamed